config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'developers#index'
+  resources :developers
+
+  post '/developers/search', to: 'developers#search'
```

Comment:
Why post?
It doesn't make sense from the Rest API point view.

I see you copied Thienhv's code.
But I didn't point out all the mistakes, so you made the same mistakes.
Please write the code yourself.

Reply:
That mean that I must not use post method. I must use resources route.
I only see how to create route and form of Thienhv's code. But I understand it. :)